x-pack/test/api_integration/apis/uptime/rest/index.ts

```diff
@@ -45,10 +45,11 @@ export default function({ getService, loadTestFile }: FtrProviderContext) {
       loadTestFile(require.resolve('./telemetry_collectors'));
     });
     describe('with real-world data', () => {
-      before('load heartbeat data', async () => await esArchiver.load('uptime/full_heartbeat'));
-      after('unload', async () => await esArchiver.unload('uptime/full_heartbeat'));
+      beforeEach('load heartbeat data', async () => await esArchiver.load('uptime/full_heartbeat'));
+      afterEach('unload', async () => await esArchiver.unload('uptime/full_heartbeat'));
```

Comment:
We aren't manipulating any data, so are you sure,it's needed to load/reload in beforEach?

Reply:
It does seem to fail without the `each`.